packages/data/src/getVariantDescriptions.ts

```diff
@@ -138,6 +143,17 @@ export function getVariantsForEachSpecialSuitCombination(
           continue;
         }
 
+        // Prism and Rainbow require 2 clueable suits, else they are just ambiguous red
+        if (
+          numSuits === 3 &&
+          ((SUITS_THAT_REQUIRE_TWO_CLUEABLE_SUITS.has(suit.name) &&
+            (suit2.noClueColors === true || suit2.allClueColors === true)) ||
+            ((suit.noClueColors === true || suit.allClueColors === true) &&
```

Comment:
From what I can tell, these should be booleans, so I don't get why the [`strict-boolean-expressions`](https://typescript-eslint.io/rules/strict-boolean-expressions/) lint was going off.

https://github.com/Hanabi-Live/hanabi-live/blob/ae74f7fbae505f0309f680e37980f043b0d79975/packages/data/src/types/Suit.ts#L16-L17